tony-core/src/main/java/com/linkedin/tony/util/HistoryFileUtils.java

```diff
@@ -15,13 +21,45 @@
 public class HistoryFileUtils {
   private static final Log LOG = LogFactory.getLog(HistoryFileUtils.class);
 
-  public static void createHistoryFile(FileSystem fs, TonyJobMetadata metadata, Path jobDir) {
+  public static void createHistoryFile(FileSystem fs, TonyJobMetadata metadata, Path jobDir, Queue<Event> eventQueue, int numRetries) {
```

Comment:
Instead of static utils, this could be a HistoryWriter class. Please add comments for the class and methods.

Reply:
I put the logic in `EventHandler` class now. See https://github.com/linkedin/TonY/pull/117/files#diff-f54d0d666f357be10268a94d9a9384a7 for more details.